src/components/create-job-form-inputs.tsx

```diff
@@ -73,9 +68,7 @@ export function CreateJobInputs(
             break;
           case 'outputFormats':
             // If no environment is selected, do not display output formats.
-            if (
-              (field as ICreateJobOutputFormatsField).environment === ''
-            ) {
+            if ((field as ICreateJobOutputFormatsField).environment === '') {
```

Comment:
`eslint`/`prettier` tends to prefer the multi-line format, I guess because it's 1985 and everyone only has 80-column displays